src/annotator/anchoring/pdf.js

```diff
@@ -448,55 +480,51 @@ export function anchor(root, selectors) {
  * @param {HTMLElement} root - The root element
  * @param {Range} range
  */
-export function describe(root, range) {
-  const normalizedRange = new BrowserRange(range).normalize();
+export async function describe(root, range) {
+  // "Shrink" the range so that the start and endpoints are at offsets within
+  // text nodes rather than any containing nodes.
+  try {
+    range = TextRange.fromRange(range).toRange();
```

Comment:
In this variant, the `range` param to the function gets mutated, whereas before the "shrunk" or otherwise-"normalized" range is assigned to a new reference. Is there any possible downside here?

Reply:
Not that I can see. The rest of the function should use the shrunk version of the range.

Reply:
I was more thinking about the about calling code having its `range` mutated unexpectedly, but, eh, it's almost assuredly a non-thing.